test/unit/sanitizer/wrap.js

```diff
@@ -94,8 +94,8 @@ module.exports.tests.latitude_negative = function(test, common) {
     t.end();
   });
   test('positive latitude wrapping - 721 degrees', function (t) {
```

Comment:
thanks, can you please change the comment too, it should read `negative latitude wrapping - 721 degrees`

Reply:
I'll update the gist